### PR TITLE
Add ABTraceTogether (Alberta, Canada)

### PR DESCRIPTION
**Pull request by forensicmike**

### CODE REVIEW

File: ca.ab.gov.ahs.contacttracing/1.0.0/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+<string>ABTraceTogether exchanges Bluetooth signals with nearby phones running the same app. These signals contain an anonymised ID, which is encrypted and changes continually to ensure your privacy.</string>
+```
+
+## Media
+```xml
+<key>NSPhotoLibraryUsageDescription</key>
+<string>Grant ABTraceTogether permissions to access your photo library if you would like to upload a photo as part of a support request</string>
+<key>NSCameraUsageDescription</key>
+<string>Grant ABTraceTogether permissions to access your camera if you would like to upload a photo as part of a support request</string>
+```
```
Comment (ivRodriguezCA) [inline ca.ab.gov.ahs.contacttracing/1.0.0/README.md]: To keep it inline with the other pages, do you mind dividing these into:

## PhotoLibrary
```xml
<key>NSPhotoLibraryUsageDescription</key>
<string>Grant ABTraceTogether permissions to access your photo library if you would like to upload a photo as part of a support request</string>
```

## Camera
```xml
<key>NSCameraUsageDescription</key>
<string>Grant ABTraceTogether permissions to access your camera if you would like to upload a photo as part of a support request</string>
```

**Review by ivRodriguezCA — CHANGES_REQUESTED** [review]
Thanks for contributing, Mike!

If you were so kind as to update the details I'd really appreciate it.

**Comment by forensicmike** [comment]
Done :)

**Comment by ivRodriguezCA** [comment]
Thanks gain, Mike! 🙏🏼

**Pull request merged**